docs/marketplace/dev/feature-walkthroughs/202108.0/persistence-acl-feature-walkthrough/configuration.md

```diff
@@ -0,0 +1,276 @@
+![Configuration entity relation diagram](https://confluence-connect.gliffy.net/embed/image/f2309504-8638-419d-abf9-783bc45c8792.png?utm_medium=live&utm_source=custom)
+The configuration, unlike the rule, is common to the system.
+The main configuration object of the module is `\Generated\Shared\Transfer\AclEntityMetadataConfigTransfer`.
+It's available for extension through the plugin system.
+All you need is create a plugin and implement `\Spryker\Zed\AclEntityExtension\Dependency\Plugin\AclEntityMetadataConfigExpanderPluginInterface`.
+
+`\Generated\Shared\Transfer\AclEntityMetadataConfigTransfer`
+
+| property | type | description |
+|-----|-----|-----|
+| aclEntityMetadataCollection | `\Generated\Shared\Transfer\AclEntityMetadataCollectionTransfer` | The collection of configurations for different entities. |                                                                                                                           | 
+| aclEntityAllowList | string[] | The set of fully qualified classes that this module does not apply to (even if the user has rules for an entity that is in the allow list). |
+
+`\Generated\Shared\Transfer\AclEntityMetadataCollectionTransfer`
+
+| property | type | description |
+|-----|-----|-----|
+| collection | `\Generated\Shared\Transfer\AclEntityMetadataTransfer[]` | The set of configurations for the models. |
+
+`\Generated\Shared\Transfer\AclEntityMetadataTransfer`
+
+| property | type | description |
+|-----|-----|-----|
+| parent | `\Generated\Shared\Transfer\AclEntityParentMetadataTransfer` | This property is used to configure inheritance. Required for entity which has rules with [Inherited scope](rules-and-scopes/inherited-scope.html), or for [Composite entity](rules-and-scopes/composite-entity.html) (See [Inherited scope vs Composite entity](rules-and-scopes/composite-entity.html#inherited-scope-vs-composite-entity)). |
+| entityName | string | Fully qualified class name of configured entity (Propel Entity). |
+| hasSegmentTable | bool | Sets if configured entity supports segmentation (see [Segment scope documentation](rules-and-scopes/segment-scope.htmlSets the default binary access mask)). |
+| defaultGlobalOperationMask | int | Sets the default binary access mask (see [Execution flow](execution-flow.html) documentation). |
+| isSubentity | bool | Indicates whether the configured entity is part of a composite object (see [Composite entity](rules-and-scopes/composite-entity.html) documentation). |
+
+`\Generated\Shared\Transfer\AclEntityParentMetadataTransfer`
+
+| property | type | description |
+|-----|-----|-----|
+| connection | `\Generated\Shared\Transfer\AclEntityParentConnectionMetadata` | This property is used to set up the relationship between the current class and the parent. |
+| entityName | string | Fully qualified class name of parent entity |
+
+Sometimes the links between the child and parent tables established not through foreign keys, but using so-called "reference columns".
```

Comment:
I think this part should go under `\Generated\Shared\Transfer\AclEntityParentConnectionMetadataTransfer`?